tests/test_client.py

```diff
@@ -114,10 +114,16 @@ def test_url(site_config_client):
 
 
 def test_list_sites(site_config_client, requests_mock):
```

Comment:
Just in case we forgot to add the client token.

```suggestion
def test_client_has_token(site_config_client):
    assert site_config_client.api_token, 'Client should have an API token'


def test_list_sites(site_config_client, requests_mock):
```